src/main/java/uk/gov/ons/ctp/response/collection/exercise/representation/CollectionExerciseDTO.java

```diff
@@ -70,6 +70,8 @@
 
   private Boolean deleted;
 
+  private SampleUnitValidationErrorDTO[] validationErrors;
```

Comment:
Lists are easier to work with than arrays, also used for caseTypes https://github.com/ONSdigital/rm-collectionexercisesvc-api/pull/10/files#diff-896f7d186ede01b90cee7e3076299b31R57

Reply:
I'd question that - since Java 8 it's much the same. I find the syntax for instantiating arrays is more succinct.  However I agree we should be consistent, hadn't spotted the List<> up there :-)